src/users/users.service.ts

```
@@ -61,4 +62,35 @@ export class UsersService {
 
     return users;
   }
+
+  isOtpVerified(user: User) {
+    const latestVerifyUser = user.verifyUser
+      .getItems()
+      .sort((a, b) => b.createdAt.getTime() - a.createdAt.getTime())[0];
+
+    if (!latestVerifyUser) {
+      throw new Error("No verification record found for this user.");
+    }
```

Do you need to throw an error here? Boolean return should be fine